FirebaseSessions/Tests/Unit/Mocks/ShadowDate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ShadowDate.swift
+//  Pods
+//
+//  Created by Leo Zhan on 2022-10-17.
+//
```

Comment:
We'll wanna put the Google license header here. You can configure Xcode to add it automatically by adding a file to `/Users/<YOURUSER>/Library/Developer/Xcode/UserData`. I'll send you the file in chat